frontend/src/ManageOrgUsers/ManageOrgUsers.jsx

```diff
@@ -17,11 +17,77 @@ class ManageOrgUsers extends React.Component {
       showNewUserForm: false,
       creatingUser: false,
       newUser: {},
+      role: '',
       idChangingRole: null,
       archivingUser: null,
+      fields: {},
+      errors: {},
     };
   }
 
+  handleValidation() {
+    let fields = this.state.fields;
+    let errors = {};
+    let formIsValid = true;
+
+    //Name
+    if (!fields['firstName']) {
+      formIsValid = false;
+      errors['firstName'] = 'This field is required';
+    }
+
+    if (typeof fields['firstName'] !== 'undefined') {
+      if (!fields['firstName'].match(/^[a-zA-Z]+$/)) {
+        formIsValid = false;
+        errors['firstName'] = 'Only letters are allowed';
+      }
+    }
+
+    if (!fields['lastName']) {
+      formIsValid = false;
+      errors['lastName'] = 'This field is required';
+    }
+
+    if (typeof fields['lastName'] !== 'undefined') {
+      if (!fields['lastName'].match(/^[a-zA-Z]+$/)) {
+        formIsValid = false;
+        errors['lastName'] = 'Only letters are allowed';
+      }
+    }
+
+    //Email
+    if (!fields['email']) {
+      formIsValid = false;
+      errors['email'] = 'This field is required';
+    }
+
+    if (typeof fields['email'] !== 'undefined') {
+      let lastAtPos = fields['email'].lastIndexOf('@');
+      let lastDotPos = fields['email'].lastIndexOf('.');
+
+      if (
+        !(
+          lastAtPos < lastDotPos &&
+          lastAtPos > 0 &&
+          fields['email'].indexOf('@@') == -1 &&
```

Comment:
I think its `          fields['email'].indexOf('@') == -1 &&`

Reply:
@gondar00 I have changed email validation to regular expression now. Please check